source/docs/software/commandbased/command-groups.rst

```diff
@@ -102,7 +102,11 @@ As mentioned earlier, command groups are `recursively composable <https://en.wik
             new SetWristPosition(m_wrist)),
          new ScoreTube(m_wrist));
 
-   .. code-tab:: c++
+   .. code-tab:: C++ (Header)
+      #include <frc2/command/SequentialCommandGroup.h>
+      #include <frc2/command/ParallelCommandGroup.h>
+   
+   .. code-tab:: C++ (Source)
```

Comment:
This is not the intended use for the header and source directives here. Please see how includes are formatted in the differential drivetrain simulation tutorial.

Reply:
Ok, reimplemented with "..." method.  Two remaining questions:
1.) Is there a recommended way to view .rst files? -- I looked at a couple of generic rst file viewers, but they didn't display nicely at all.

2.) Do I need to create another pull request? -- I have committed and pushed changes to my forked git repository under the same branch (patch-1) as this pull-request.

Best,
David

Reply:
The build instructions will tell you how to build the docs so you can view your changes locally. 

https://docs.wpilib.org/en/stable/docs/contributing/frc-docs/build-instructions.html